client/human/HumanClientFSM.cpp

```diff
@@ -533,6 +534,20 @@ boost::statechart::result MPLobby::react(const CheckSum& e) {
     return discard_event();
 }
 
+boost::statechart::result MPLobby::react(const ChatHistory& msg) {
+    InfoLogger(FSM) << "(HumanClientFSM) ChatHistory.";
```

Comment:
Please do not use `info` severity for detailed log messages (similar uses in this file are `trace`)

Reply:
It was for a test.